src/api_client/voctoweb_client.py

```diff
@@ -190,12 +192,16 @@ def create_event(self):
         try:
             # TODO make ssl verify a config option
             # r = requests.post(url, headers=headers, data=json.dumps(payload), verify=False)
-            r = requests.post(url, headers=headers, data=json.dumps(payload))
+            if self.t.voctoweb_event_id:
+                r = requests.patch(url, headers=headers, data=json.dumps(payload))
+            else:
+                r = requests.post(url, headers=headers, data=json.dumps(payload))
+
```

Comment:
@derpeter Der Kern dieses PRs ist sind die oben stehenden Zeilen: Wenn wir schon ne ID für die Entity in Voctoweb kennen dann patch anstatt post.